app/models/flood_risk_engine/address_search.rb

```diff
@@ -0,0 +1,5 @@
+module FloodRiskEngine
+  class AddressSearch < ActiveRecord::Base
```

Comment:
Yes, you are right. Clearing address_search will mess up the back behaviour. Perhaps we should leave this until the next address builder in the process needs to be built.